include/user_settings.h

```diff
@@ -122,9 +126,7 @@
 #endif
 
 /* Disables - For minimum wolfCrypt build */
-#define NO_AES
 #define NO_CMAC
-#define NO_HMAC
```

Comment:
Use #ifndef WOLFTPM instead of removing the line

Reply:
thanks, will do